### DISCUSSION TITLE
Error: TASK [k3s/master : Copy vip manifest to first master]

### DISCUSSION THREAD

**Discussion by gcraenen** [techno-tim/k3s-ansible#164] [Questions] [answered]
Scripts run without error and k8s cluster is provisioned, but:

## Current Behavior

TASK [k3s/master : Copy vip manifest to first master] *************************************************************************************
Saturday 12 November 2022  15:28:54 +0000 (0:00:00.848)       0:00:27.347 ***** 
[WARNING]: an unexpected error occurred during Jinja2 environment setup: unable to locate collection ansible.utils
skipping: [k8s-master02.calmus.one]
fatal: [k8s-master01.calmus.one]: FAILED! => changed=false 
  msg: |-
    AnsibleError: template error while templating string: unable to locate collection ansible.utils. String: apiVersion: apps/v1
    kind: DaemonSet
    metadata:
      name: kube-vip-ds
      namespace: kube-system
    spec:
      selector:
        matchLabels:
          name: kube-vip-ds
      template:
        metadata:
          labels:
            name: kube-vip-ds
        spec:
          affinity:
            nodeAffinity:
              requiredDuringSchedulingIgnoredDuringExecution:
                nodeSelectorTerms:
                - matchExpressions:
                  - key: node-role.kubernetes.io/master
                    operator: Exists
                - matchExpressions:
                  - key: node-role.kubernetes.io/control-plane
                    operator: Exists
          containers:
          - args:
            - manager
            env:
            - name: vip_arp
              value: "true"
            - name: port
              value: "6443"
            - name: vip_interface
              value: {{ flannel_iface }}
            - name: vip_cidr
              value: "{{ apiserver_endpoint | ansible.utils.ipsubnet | ansible.utils.ipaddr('prefix') }}"
            - name: cp_enable
              value: "true"
            - name: cp_namespace
              value: kube-system
            - name: vip_ddns
              value: "false"
            - name: svc_enable
              value: "false"
            - name: vip_leaderelection
              value: "true"
            - name: vip_leaseduration
              value: "15"
            - name: vip_renewdeadline
              value: "10"
            - name: vip_retryperiod
              value: "2"
            - name: address
              value: {{ apiserver_endpoint }}
            image: ghcr.io/kube-vip/kube-vip:{{ kube_vip_tag_version }}
            imagePullPolicy: Always
            name: kube-vip
            resources: {}
            securityContext:
              capabilities:
                add:
                - NET_ADMIN
                - NET_RAW
                - SYS_TIME
          hostNetwork: true
          serviceAccountName: kube-vip
          tolerations:
          - effect: NoSchedule
            operator: Exists
          - effect: NoExecute
            operator: Exists
      updateStrategy: {}
    status:
      currentNumberScheduled: 0
      desiredNumberScheduled: 0
      numberMisscheduled: 0
      numberReady: 0
skipping: [k8s-master03.calmus.one]

**Comment by gcraenen** [comment, marked as answer]
Found it. Galaxy install was not ok and had to be reinstalled.

**Reply by nsadlo** [reply]
For others that do not know what that means.

That means run "ansible-galaxy collection install ansible.utils" in your terminal to install the ansible utils.
